File: app/src/main/java/com/penkzhou/projects/android/playground/PlaygroundApplication.java
package com.penkzhou.projects.android.playground;

import android.app.Application;
import android.content.Context;

import com.facebook.stetho.Stetho;

public class PlaygroundApplication extends Application {
    private static Context context;

    public void onCreate() {
        super.onCreate();
        PlaygroundApplication.context = getApplicationContext();
        Stetho.initializeWithDefaults(this);
    }

    public static Context getAppContext() {
        return PlaygroundApplication.context;
    }
}

File: app/src/main/java/com/penkzhou/projects/android/playground/PlaygroundApplication.kt
package com.penkzhou.projects.android.playground

import android.app.Application
import android.content.Context

import com.facebook.stetho.Stetho

class PlaygroundApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        PlaygroundApplication.appContext = applicationContext
        Stetho.initializeWithDefaults(this)
    }

    companion object {
        var appContext: Context? = null
            private set
    }
}
